### PR TITLE
Added recipe for pychef

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@jakirkham :shipit:  ?

File: recipes/pychef/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - requests
+    - setuptools
+    - six
```
Comment (jakirkham) [inline recipes/pychef/meta.yaml]: Do we need `six` at build time or only run time?

File: recipes/pychef/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - chef
+
+about:
+  home: https://github.com/coderanger/pychef
+  license: Apache 2.0
+  license_family: Apache
+  # license_file: LICENSE
```
Comment (jakirkham) [inline recipes/pychef/meta.yaml]: Could we please raise an issue or PR upstream? Also please xref it back here when done.

Comment (jakirkham) [inline recipes/pychef/meta.yaml]: Might also mention [this]( https://github.com/coderanger/pychef/blob/45238431f6d34dc2daffd3ebf1e88ea563243353/setup.py#L15 ) to them.

Comment (mariusvniekerk) [inline recipes/pychef/meta.yaml]: That is rather odd.

Comment (mariusvniekerk) [inline recipes/pychef/meta.yaml]: https://github.com/coderanger/pychef/pull/71

File: recipes/pychef/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  imports:
+    - chef
+
+about:
+  home: https://github.com/coderanger/pychef
+  license: Apache 2.0
+  license_family: Apache
+  # TODO: Reintroduce once https://github.com/coderanger/pychef/pull/71 lands
+  # license_file: LICENSE
```
Comment (jakirkham) [inline recipes/pychef/meta.yaml]: 👍

**Comment by jakirkham** [comment]
Thanks @mariusvniekerk.

**Pull request merged**